nscplugin/src/main/scala/scala/scalanative/nscplugin/NirGenName.scala

```diff
@@ -78,7 +78,12 @@ trait NirGenName { self: NirGenPhase =>
     if (sym == String_+) {
       genMethodName(StringConcatMethod)
     } else if (sym.owner.isExternModule) {
-      owner member id tag "extern"
+      if (sym.isSetter) {
+        val id0 = sym.name.dropSetter.decoded.toString
```

Comment:
This drops the `_=` part of the name and then decodes the name as done by `nativeIdOf`.